File: 0513/513.go
func findBottomLeftTreeValue(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := findBottomLeftTreeValue(root.Left)
	r := findBottomLeftTreeValue(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 0513/513.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var findBottomLeftTreeValue = function(root) {
    if (!root) return 0;
    return 1 + Math.max(findBottomLeftTreeValue(root.left), findBottomLeftTreeValue(root.right));
};